examples/multiline_logs/values_containerd.yaml

```
@@ -0,0 +1,168 @@
+fluent-bit:
+  luaScripts:
+    parse_logs.lua: |
+      local function split(s, delimiter)
+        result = {};
+        for match in (s..delimiter):gmatch('(.-)'..delimiter) do
+            table.insert(result, match);
+        end
+        return result;
+      end
+
+      function get_log_content(line)
+        -- remove elements specific containerd log format and get log as single string
+        table.remove(line, 1) -- remove date
+        table.remove(line, 1) -- remove stream
+        table.remove(line, 1) -- remove log tag
+        return table.concat(line, " ")
+      end
+
+      function adjust_first_line(record, first_line)
+        -- adjust the first line to containerd format, it comes without date, stream and logtag
+        first_line = 'date' .. ' ' .. record['stream'] .. ' ' .. record['logtag'] .. ' ' .. first_line
```

Shouldn't this be `record['date']' instead? Or whatever other field holds it.
```suggestion
        first_line = record['date'] .. ' ' .. record['stream'] .. ' ' .. record['logtag'] .. ' ' .. first_line
```

---

`date` key is not available, `time` key is used as timestamp in this configuration, see [this](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2009/files#diff-3ca3694d28ef9cd9ac8d31a0c403e61d5a475afa3061478ddc7328637dd5ee82R156) line. 
`date` string is used only to have proper format of log, it could be replaced with any string.
I changed `date` to `fake-date` and I added comment explaining why it is used.